api/secure_variables.go

```diff
@@ -445,3 +462,83 @@ func generateUnexpectedResponseCodeError(resp *http.Response) error {
 	resp.Body.Close()
 	return fmt.Errorf("Unexpected response code: %d (%s)", resp.StatusCode, buf.Bytes())
 }
+
+type VarUI interface {
+	GetConcurrentUI() cli.ConcurrentUi
+	Colorize() *colorstring.Colorize
+}
```

Comment:
None of this presentation-level stuff has callers in this PR. And I don't think it belongs here rather in the `api` package rather than our `command` package in any case (it won't build, either).